Index: index.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print('hello world')\nprint('hello world111')\nprint('hello worldsss')\nprint('hello worldzzzz')\nprint('hello worldsss123')\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- index.py	(revision 05393f7011c5fbd0febf087c60a2d243cb7c2a90)
+++ index.py	(date 1604525075736)
@@ -3,3 +3,6 @@
 print('hello worldsss')
 print('hello worldzzzz')
 print('hello worldsss123')
+print('hello worldsss1234')
+print('hello worldsss12345')
+print('hello worldsss12367')
\ No newline at end of file
